static/templates/filings/candidate.hbs

```diff
@@ -56,14 +56,28 @@
       <td class="panel__term">Ending cash on hand</td>
       <td class="panel__data">{{{currency cash_on_hand_end_period}}}</td>
     </tr>
+    {{#if net_contributions_period}}
     <tr>
       <td class="panel__term">Net contributions</td>
       <td class="panel__data">{{{currency net_contributions_period}}}</td>
     </tr>
+    {{else}}
+    <tr>
+      <td class="panel__term">Net contributions</td>
+      <td class="panel__data">{{{currency net_contributions_cycle_to_date}}}</td>
+    </tr>
+    {{/if}}
+    {{#if net_operation_expenditures_period}}
```

Comment:
As for your second point, i don't understand what's different. Isn't this report view showing all financial summary information from the report? If it's not, this is where it should be.